thrimshim/thrimshim/main.py

```diff
@@ -147,14 +130,46 @@ def update_row(ident, new_row):
 			sql.SQL("{} = {}").format(
 				sql.Identifier(column), sql.Placeholder(column),
 			) for column in new_row.keys()
-		))
+	))
 	result = database.query(conn, build_query, id=ident, **new_row)
 	if result.rowcount != 1:
 		return 'Video likely already published', 403	
 			
 	logging.info('Row {} updated to state {}'.format(ident, new_row['state']))
 	return ''
 
+
+@app.route('/thrimshim/manual-link/<uuid:ident>', methods=['POST'])
+def manual_link(ident):
+	"""Manually set a video_link"""
+	link = flask.request.json
+	conn = app.db_manager.get_conn()
+	results = database.query(conn, """
+		UPDATE events 
+		SET state='DONE', upload_location = 'Manual', video_link = %s
```

Comment:
let's make `manual` lowercase, just because having one consistent rule is easier to remember